dbms/src/DataTypes/DataTypesDecimal.h

```diff
@@ -243,15 +75,17 @@ inline UInt32 getDecimalScale(const IDataType & data_type, UInt32 default_value
     return default_value;
 }
 
-///
+template <typename T>
+inline UInt32 getDecimalScale(const DataTypeDecimal<T> & data_type)
+{
+    return data_type.getScale();
+}
 
-template <typename DataType> constexpr bool IsDataTypeDecimal = false;
```

Comment:
It looks like it depends on includes now. What if I do not include DataTypesDecimal.h in some cpp (just declare it with using template)? Would it return true?
I don't think it's a good idea to split this blok of code between files.

Reply:
yep, that could be an issue

Reply:
fixed